package.json

```diff
@@ -15,12 +20,18 @@
     "JavaScript",
     "Node",
     "Browser",
-    "Isomorphic"
+    "Universal"
```

Comment:
I'd leave "Isomorphic" as well until the mindshare has shifted completely to "Universal" -- that will take a while... maybe 2 years+